app/controllers/posts_controller.rb

```diff
@@ -10,6 +10,7 @@ def index
   def show
     @current_user = current_user
     @post = Post.find(params[:id])
+    @comms = @post.comments.includes(:author)
```

Comment:
Awesome! N+1 problems were solved correctly 🔥

Reply:
Thank you for the review, Mateo.